code/surface_area/compute_surface_area.py

```diff
@@ -0,0 +1,172 @@
+import argparse
+from pathlib import Path
+from typing import Iterable
+
+from niwrap import workbench as wb
+from styxdocker import DockerRunner
+
+from validate_surface_files import validate_output_file_data
+from utils import find_surface_files, get_map_info
+
+my_runner = DockerRunner()
+my_runner.data_dir = Path(__file__).parent / "../../share/Inputs"
+
+
+def compute_surface_area(input_gifti: Path, output_metric: Path) -> None:
+    """
+    Compute the surface area of a brain surface mesh using niwrap.
+
+    """
+    # test if file exists
+    if not input_gifti.exists():
+        print(f"✗ Input file does not exist: {input_gifti}")
+        return
```

Comment:
This is good, especially for large packages, and no need to change it - within this script it isn't strictly necessary since we are using `glob` to find files in the directory (which exists).